app/assets/images/payment_icons/nationaleentertainmentcard.svg

```diff
@@ -0,0 +1,22 @@
+<svg fill="none" viewBox="0 0 38 24" width="38" height="24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
Same comments apply about the `id` attributes.
Also the borders have some pixels issues as well as the radius
[![alt](https://screenshot.click/16-05-nug4x-l7df1.png)](https://screenshot.click/16-05-nug4x-l7df1.png)

You can follow this [example icon](https://github.com/activemerchant/payment_icons/blob/master/app/assets/images/payment_icons/visa.svg) as a reference.

Reply:
Fixed pixel issues. I left the ID's that are used as a reference for the mask/gradients.